packages/@aws-cdk/aws-apigateway/lib/model.ts

```diff
@@ -1,46 +1,153 @@
-export interface IModel {
-    readonly modelId: string;
+import { Construct, IResource, Resource } from '@aws-cdk/core';
+import { CfnModel, CfnModelProps } from './apigateway.generated';
+import jsonSchema = require('./json-schema');
+import { IRestApi, RestApi } from './restapi';
+
+export interface IModel extends IResource {
+  /**
+   * A name for the model. If you don't specify a name, AWS CloudFormation
+   * generates a unique physical ID and uses that ID for the model name.
+   * For more information, @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-name.html.
+   *
+   * Important
+   * If you specify a name, you cannot perform updates that require replacement
+   * of this resource. You can perform updates that require no or some interruption.
+   * If you must replace the resource, specify a new name.
+   *
+   * @attribute
+   */
+  readonly modelId: string;
 }
 
-/**
- * Represents a reference to a REST API's Empty model, which is available
- * as part of the model collection by default. This can be used for mapping
- * JSON responses from an integration to what is returned to a client,
- * where strong typing is not required. In the absence of any defined
- * model, the Empty model will be used to return the response payload
- * unmapped.
- *
- * Definition
- * {
- *   "$schema" : "http://json-schema.org/draft-04/schema#",
- *   "title" : "Empty Schema",
- *   "type" : "object"
- * }
- *
- * @see https://docs.amazonaws.cn/en_us/apigateway/latest/developerguide/models-mappings.html#models-mappings-models
- */
-export class EmptyModel implements IModel {
-    public readonly modelId = 'Empty';
+export interface ModelOptions {
+  /**
+   * The content type for the model.
+   * @default None
```

Comment:
Can you elaborate in the docs what does it mean not to provide a content type?

Also, the value in @default should be “-“ to indicate it’s not a “real” value

Reply:
Done